sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2428,6 +2429,28 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DelayDenyAddAndCustomizePagesEnforcement
+In the SharePoint admin center, there is a Custom Scripts setting that controls whether users can run custom scripts in OneDrive. This setting will be removed on March 2024. By removing this setting, Administrators will no longer be able to control the default behavior on self service creates sites or personal sites. However, customers will still be able to temporarily enable the ability to run custom script by setting `Set-SPOTenant -DelayDenyAddAndCustomizePagesEnforcement $true`. This will keep the setting available until May 2024.
```

Comment:
I would slightly rephrase it providing less information here and leave the details to the doc you pointed out. How about
"This parameter controls how SharePoint will deal with sites where custom scripts are allowed. Possible values are:

- false (default): for site collections where administrators enabled the ability to add custom script, SharePoint will revoke that ability within 24 hours from the last time this setting was changed.
- true: all changed performed by administrators to custom script settings are preserved. When the value is set to true, a banner shows in the SharePoint tenant admin in the lists page informing that changes to custom scripts are permanent.
> [!NOTE]
> This setting affects all sites. There are no options to preserve changes to custom script settings only on specific sites
> [!NOTE]
> This parameter will be available **until May 2024**. After that time, administrators can still allow custom scripts on specific sites, but that change will be revoked automatically after up to 24 hours."

Reply:
Updated!